src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+    }
+    else{
+        emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[this.houses]);
+    }
 
-  return this;
+    return this;
 }
 
 function railroad(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
```

Comment:
I think my huge comment at the bottom avoids problems with this. As for malicious clients (i.e. Bobby) trying to force a promptWhatever to go through on not their turn, basically the solution is when a client replies, make sure that it's their turn before processing anything.